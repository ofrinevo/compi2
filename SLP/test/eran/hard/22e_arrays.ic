
class MainClass { 

	static void main (string[] args) { 

		int a = -3;
		Library.println("a = -3");
		Library.println("new int[a]");
		int[] arr = new int[a];
		Library.println("Should not print this or get here.");

	}

}
