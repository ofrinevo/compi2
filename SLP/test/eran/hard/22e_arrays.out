a = -3
new int[a]
Runtime Error: Array allocation with negative array size!
